src/s11/handlers/ddn_ack_handler.c

```diff
@@ -59,7 +59,7 @@ ddn_ack_processing(struct DDN_ACK_Q_msg *ddn_ack_msg)
 	gtpHeader.msgType =  GTP_DOWNLINK_DATA_NOTIFICATION_ACK;
 	gtpHeader.sequenceNumber = ddn_ack_msg->seq_no;
 	gtpHeader.teidPresent = true;
-	gtpHeader.teid = ddn_ack_msg->ue_idx;
+	gtpHeader.teid = ddn_ack_msg->s11_sgw_cp_teid;
 
 	g_s11_sequence++;
```

Comment:
We need sequence management API/module within GTP. Why to increase sequence number while sending Ack message ?

Reply:
Sequence number need not be incremented. Addressed in the commit https://github.com/omec-project/Nucleus/pull/15/commits/9d5ceca6255e143e79cf646ed94153e5fa0a37bf